app/routes/submissions.js

```diff
@@ -0,0 +1,105 @@
+import Route from '@ember/routing/route';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { isPresent } from '@ember/utils';
+import { startOfDay, endOfDay } from 'date-fns';
+// import CONSTANTS from 'frontend-kaleidos/config/constants';
+import parseDate from 'frontend-kaleidos/utils/parse-date-search-param';
+
+export default class SubmissionsRoute extends Route {
+  @service currentSession;
+  @service conceptStore;
+  @service store;
+  @service router;
+  @service('session') simpleAuthSession;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+      as: 'pagina',
+    },
+    size: {
+      refreshModel: true,
+      as: 'aantal',
+    },
+    sortSubmissions: {
+      refreshModel: true,
+      as: 'sorteer',
+    },
+    dateFrom: {
+      refreshModel: true,
+      as: 'van',
+    },
+    dateTo: {
+      refreshModel: true,
+      as: 'tot',
+    },
+    submitters: {
+      refreshModel: true,
+      as: 'indieners',
+    },
+  };
+
+  async beforeModel(transition) {
+    this.simpleAuthSession.requireAuthentication(transition, this.simpleAuthSession.unauthenticatedRouteName);
+
+    if (!this.currentSession.may('view-submissions')) {
+      this.router.transitionTo('cases.index');
+    }
+  }
+
+  async model(params) {
+    // *note: the cache busting delays the loading a bit, even locally with only 2 submissions it take half a second
+    const options = {
+      'filter[:has:created]': `date-added-for-cache-busting-${new Date().toISOString()}`,
+      'filter[pieces][accepted-piece][agendaitems][agenda][created-for][:has-no:agenda]': 't', // filter out submissions on closed meetings
```

Comment:
Edge case: if the last piece was removed after being accepted, an "empty" submission might perpetually remain in the list.

Adding a `:has:pieces` filter to cover this.